src/plugins/workspace/public/components/workspace_initial/workspace_initial.tsx

```diff
@@ -190,39 +240,33 @@ export const WorkspaceInitial = () => {
   return (
     <EuiPage style={{ minHeight: '100vh' }}>
       <EuiPageBody>
-        <EuiFlexGroup direction="column" alignItems="center" style={{ paddingTop: '100px' }}>
-          <EuiFlexItem grow={false}>
-            <EuiImage size="l" alt="OpenSearch" src={logos.OpenSearch.url} />
-          </EuiFlexItem>
-          <EuiSpacer />
-          <EuiFlexItem grow={false} style={{ width: '1200px' }} className="eui-displayInline">
-            <EuiPageContent
-              style={{
-                backgroundImage: `url(${backGroundUrl})`,
-                backgroundSize: '490px 380px',
-                backgroundRepeat: 'no-repeat',
-                backgroundPosition: 'right 10px top 20px',
-              }}
-            >
-              {content}
-            </EuiPageContent>
-            <EuiSpacer size="m" />
-            <EuiSmallButtonEmpty
-              iconType="gear"
-              iconSide="left"
-              flush="left"
-              href={settingsAndSetupUrl}
-              data-test-subj="workspace-initial-button-settingsAndSetup"
-            >
-              <EuiText size="s">
-                {i18n.translate('workspace.initial.button.settingsAndSetup', {
-                  defaultMessage: 'Settings and setup',
-                })}
-              </EuiText>
-            </EuiSmallButtonEmpty>
+        <EuiIcon type="logoOpenSearch" size="xl" />
+        <EuiFlexGroup
+          direction="column"
+          alignItems="center"
+          justifyContent="flexStart"
+          style={{ paddingTop: '24px' }}
+        >
```

Comment:
Instead of inline css, perhaps we can use Spacer component
```suggestion
        <EuiSpacer size="l" />
        <EuiFlexGroup
          direction="column"
          alignItems="center"
          justifyContent="flexStart"
        >
```

Reply:
updated